### PR TITLE
APR-1294 - RoATP Gateway : Register Checks

### PR DESCRIPTION
**Pull request by martin-hornsby**

### CODE REVIEW

File: src/SFA.DAS.AdminService.Web/Services/Gateway/GatewayRegisterChecksOrchestrator.cs [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+            return route;
+        }
+
+        private async Task<string> GetProviderStatus(int? providerStatusId, int? providerTypeId)
+        {
+            string status = null;
+
+            if(providerStatusId.HasValue)
+            {
+                var roatpStatues = await _roatpApiClient.GetOrganisationStatuses(providerTypeId);
```
Comment (David-B-Read) [inline src/SFA.DAS.AdminService.Web/Services/Gateway/GatewayRegisterChecksOrchestrator.cs]: Typo in var name

File: src/SFA.DAS.AdminService.Web/ViewModels/Roatp/Gateway/RoatpPageViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        public DateTime? SourcesCheckedOn { get; set; }
+
+        public string ApplyLegalName { get; set; }
+        public string ApplyProviderRoute { get; set; }
+
+        public bool RoatpUkprnOnRegister { get; set; }
+        public string RoatpProviderRoute { get; set; }
+        public DateTime? RoatpStatusDate { get; set; }
+        public string RoatpStatus { get; set; }
+        public string RoatpRemoveReason { get; set; }
```
Comment (David-B-Read) [inline src/SFA.DAS.AdminService.Web/ViewModels/Roatp/Gateway/RoatpPageViewModel.cs]: Should this be RoatpRemovedReason ?

File: src/SFA.DAS.AdminService.Web/Services/Gateway/GatewayRegisterChecksOrchestrator.cs (hunk 1 above)
Comment (martin-hornsby) [reply src/SFA.DAS.AdminService.Web/Services/Gateway/GatewayRegisterChecksOrchestrator.cs]: corrected 👍

File: src/SFA.DAS.AdminService.Web/ViewModels/Roatp/Gateway/RoatpPageViewModel.cs (hunk 2 above)
Comment (martin-hornsby) [reply src/SFA.DAS.AdminService.Web/ViewModels/Roatp/Gateway/RoatpPageViewModel.cs]: corrected 👍

**Pull request merged**